extensions/tokenizers/src/main/java/ai/djl/engine/rust/jni/JniUtils.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.engine.rust.jni;
+
+import ai.djl.engine.rust.RsNDArray;
+import ai.djl.engine.rust.RsNDManager;
+import ai.djl.engine.rust.RustLibrary;
+import ai.djl.ndarray.NDList;
+
+import java.util.List;
+
+/**
+ * A class containing utilities to interact with the Rust Engine's Java Native Interface (JNI)
+ * layer.
+ */
+@SuppressWarnings("MissingJavadocMethod")
```

Comment:
We don't really need this wrap. We can call RustLibrary directly.

Reply:
Done